infra/README.md

```diff
@@ -179,7 +185,8 @@ You can now install Hail:
 
   ```
   cd $HAIL
-  export HAIL_CI_UTILS_IMAGE=gcr.io/<gcp-project>/ci-utils:latest
+  export HAIL_DOCKER_PREFIX=gcr.io/<gcp-project>
```

Comment:
Do you want to put another comment here (and below) about using the AR?

Reply:
Added!